source/common/common/mem_block_builder.h

```diff
@@ -31,8 +31,8 @@ template <class T> class MemBlockBuilder {
 
   /**
    * Populates (or repopulates) the MemBlockBuilder to make it the specified
-   * capacity. This does not have resize semantics; when populate() is called any
-   * previous contents are erased.
+   * capacity. This does not have resize semantics; when populate() is called
```

Comment:
Total bikeshedding nit: the reason I don't like populate is it sounds like we are actually filling it with something, which we aren't. Perhaps `setCapacity(...)`? Up to you.

Reply:
OK happy to clean this up now; I'll got with setCapacity.